### DISCUSSION TITLE
Edit and RUN your code from GitHub online in seconds

### DISCUSSION THREAD

**Discussion by DenverCoder1** [EddieHubCommunity/support#2190] [Tips and Tricks]
We all know about [github1s](https://github.com/conwnet/github1s) and the new [github.dev](https://twitter.com/github/status/1425505817827151872) shortcut, but no one's talking about this shortcut which was also just released that not only lets you view and edit your GitHub code in your browser, but also lets you **build and run** your code with a terminal!

~~You can now create repositories, fork repositories, commit & push code directly from Anyfiddle's interface.~~

~~You can also start an Anyfiddle project from any public Github repo by changing github to githubaf in the url.~~

~~You can check out how it works in my latest video!~~

https://www.youtube.com/watch?v=0YLTILjOxIE

## UPDATE

Anyfiddle is no longer operational as of Jan 14, 2022. I suggest checking out [Gitpod.io](https://gitpod.io) instead

https://www.youtube.com/watch?v=Mt_Bsj6K9Lw

[![Run Open Source Code in Seconds with GitPod](https://freshidea.com/jonah/app/youtube-card/?id=Mt_Bsj6K9Lw)](https://www.youtube.com/watch?v=Mt_Bsj6K9Lw "Run Open Source Code in Seconds with GitPod")

**Comment by sowmya-hub** [comment]
I got to know about Gitpod through Eddie's youtube channel when he made a video of it with Pauline, second it to speed up coding!

**Reply by eddiejaoude** [reply]
Thanks for letting me know 👍  Gitpod is great!

**Reply by shashreesamuel** [reply]
Eddie really has quality content on his Youtube, I learnt to code a simple discord bot while following his youtube tutorials
